### PR TITLE
blog en: add 7.1.1 entry

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: en/_posts/2018-01-29-groonga-7.1.1.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+``
+
+### [logical_range_filter](/docs/reference/commands/logical_range_filter.html#post-filter) Added a new filter timing.
+
+You can executed `filter` again after `filtered` stage generated columns are generated.
+
+### Conclusion
+
+See [Release 7.1.1 2017-12-29](/docs/news.html#release-7.1.1) about detailed changes since 7.1.0
+
```
Comment (kenhys) [inline en/_posts/2018-01-29-groonga-7.1.1.md]: It should be 2018-01-29. @komainu8

File: en/_posts/2018-01-29-groonga-7.1.1.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+```text
+ % grndb recover --force-lock-clear DB_PATH
+```
+
+### [load](/docs/reference/commands/load.html) added surrogate pairs support in escape syntax.
+
+You can use surrogate pairs in escape syntax in [load](/docs/reference/commands/load.html).
+For example, ``\\uD83C\\uDF7A`` is processed as ``🍺``."
+
```
Comment (kenhys) [inline en/_posts/2018-01-29-groonga-7.1.1.md]: It seems that there is a garbage " at the end.

**Review by kenhys — CHANGES_REQUESTED** [review]
Could you fix it?

**Comment by komainu8** [comment]
@kenhys Thank you for your comments. I've fixed them.

**Comment by kenhys** [comment]
LGTM

**Pull request merged**